### ISSUE TITLE
CS-GO bugged in Ubuntu 17

### ISSUE THREAD

**Issue by ilPeolo** [ValveSoftware/steam-for-linux#4931]
#### Your system information
![screenshot from 2017-04-17 23-57-38](https://cloud.githubusercontent.com/assets/22934210/25125457/3187fae2-242f-11e7-9b69-49cfd5873658.png)

* Steam client version: 23 march 
* Distribution Ubuntu 17.04 
* Opted into Steam client beta?: No
* Have you checked for system updates?: Yes

#### Please describe your issue in as much detail as possible:
When I started a game after upgrading to 17.04 from 16.10 everything looked black (see the picture). I tried to reinstall the game but it didn't help
#### Steps for reproducing this issue:
Just start a game of whatever type in ubuntu 17.04.

https://gist.github.com/ilPeolo/6031ed15076ae4886635f926fa391cbf

**Comment by kisak-valve** [comment]
Hello @ilPeolo, please copy your system information from steam (```Steam``` -> ```Help``` -> ```System Information```) and put it in a [gist](https://gist.github.com/), then include a link to the gist in this issue report.

**Comment by ilPeolo** [comment]
I updated the issue report with the system information

**Comment by kisak-valve** [comment]
The most common culprit of this kind of symptom is missing s3tc support. Can you check if you have the 64 bit variant of http://packages.ubuntu.com/zesty/libtxc-dxtn-s2tc?

**Comment by ilPeolo** [comment]
I don't have the libtxc-dxtn-s2tc installed. And I don't know what is the 64 bit variant is

**Comment by kisak-valve** [comment]
Since you are running 64 bit Ubuntu, the 64 bit variant is the default (the 32 bit variant would be libtxc-dxtn-s2tc:i386)  Pease run `sudo apt install libtxc-dxtn-s2tc0` in a terminal and check if it helps.

**Comment by ilPeolo** [comment]
It worked! thanks for the help :+1:

**Comment by kisak-valve** [comment]
You may also need to install libtxc-dxtn-s2tc0:i386 at some point for 32 bit games. Closing as this was a system config issue..

**Comment by kisak-valve** [comment]
Hello @ilPeolo, I took another look at this now that I have an ubuntu 17.04 test box and the apt command I suggested before may have removed the steam:i386 package, which can give you a headache down the road.

`sudo apt-get install libtxc-dxtn-s2tc:i386 libtxc-dxtn-s2tc:amd64` should work as the expected workaround.

This issue is being tracked at https://bugs.launchpad.net/ubuntu/+source/steam/+bug/1684755.
